deployment/build_iati_standard.sh

```diff
@@ -51,14 +62,42 @@ echo -e "UPDATED DOCS & TEMPLATES \n\n"
 # Regenerate all versions of the sites, saving HTML outputs in '_build/dirhtml'
 for f in 2.01 2.02 2.03 1.05 1.04; do
     cd $f
-    echo -e "NOW IN FOLDER: $f \n\n"
+    site_folder="${f//.}"
+    echo -e "NOW IN FOLDER: $PWD \n\n"
+
+
+
+    # Ensure that directories are up to date and clean of changes
+    git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
+    git fetch origin
+
+    git reset --hard HEAD^1
+    git clean -fd
 
     # Ensure code and submodules are up-to-date with origin
-    git pull
-    git submodule update
+    if [ "$CHECKOUT" = true ] ; then
+        git checkout version-$f
+        git pull origin version-$f
+    else
+        git pull
+    fi
+
+    git submodule deinit --all -f
+    git submodule update --init --recursive
     echo -e "DONE CODE PULLS: $f \n\n"
 
     # Set-up the environment and repository dependencies
+    # Remove existing pyenv to replace with new
+    if [ -d "pyenv/" ]; then
+        echo "REMOVING EXISTING PYENV" 
+        rm -Rf pyenv/
+    fi
+
+    if [ "$PYTHON_2" = true ]; then    
```

Comment:
this is good, do we want to keep it though? should we not be a bit more annoying and just force the users to have py3?

Reply:
this flag can be removed when py3 upgrades have been merged and deployed, otherwise not needed!